auraescript/Cargo.toml

```diff
@@ -1,69 +1,26 @@
-# ---------------------------------------------------------------------------- #
-#             Apache 2.0 License Copyright © 2022 The Aurae Authors            #
-#                                                                              #
-#                +--------------------------------------------+                #
-#                |   █████╗ ██╗   ██╗██████╗  █████╗ ███████╗ |                #
-#                |  ██╔══██╗██║   ██║██╔══██╗██╔══██╗██╔════╝ |                #
-#                |  ███████║██║   ██║██████╔╝███████║█████╗   |                #
-#                |  ██╔══██║██║   ██║██╔══██╗██╔══██║██╔══╝   |                #
-#                |  ██║  ██║╚██████╔╝██║  ██║██║  ██║███████╗ |                #
-#                |  ╚═╝  ╚═╝ ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝ |                #
-#                +--------------------------------------------+                #
-#                                                                              #
-#                         Distributed Systems Runtime                          #
-#                                                                              #
-# ---------------------------------------------------------------------------- #
-#                                                                              #
-#   Licensed under the Apache License, Version 2.0 (the "License");            #
-#   you may not use this file except in compliance with the License.           #
-#   You may obtain a copy of the License at                                    #
-#                                                                              #
-#       http://www.apache.org/licenses/LICENSE-2.0                             #
-#                                                                              #
-#   Unless required by applicable law or agreed to in writing, software        #
-#   distributed under the License is distributed on an "AS IS" BASIS,          #
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
-#   See the License for the specific language governing permissions and        #
-#   limitations under the License.                                             #                                                                             #
-#                                                                              #
-# ---------------------------------------------------------------------------- #
-
 [package]
 name = "auraescript"
 version = "0.1.0"
 edition = "2021"
-authors = ["The Aurae Authors", "Kris Nóva <kris@nivenly.com>"]
+thors = ["The Aurae Authors", "Kris Nóva <kris@nivenly.com>"]
```

Comment:
"we're still worthy!" but i think this should be "authors"

Reply:
of course I messed up copy pasting, lol